internal/csi-common/utils.go

```diff
@@ -222,18 +223,23 @@ func getReqID(req interface{}) string {
 		reqID = r.GetGroupSnapshotId()
 
 	// Replication
-	case *replication.EnableVolumeReplicationRequest:
-		reqID = GetIDFromReplication(r)
-	case *replication.DisableVolumeReplicationRequest:
-		reqID = GetIDFromReplication(r)
-	case *replication.PromoteVolumeRequest:
-		reqID = GetIDFromReplication(r)
-	case *replication.DemoteVolumeRequest:
-		reqID = GetIDFromReplication(r)
-	case *replication.ResyncVolumeRequest:
-		reqID = GetIDFromReplication(r)
-	case *replication.GetVolumeReplicationInfoRequest:
+	case *replication.EnableVolumeReplicationRequest,
```

Comment:
As this is the pattern followed for all other RPC calls, lets keep it as it is unless some linter complains about it.

Reply:
Yes, the change is because of the `gocyclo` complain from the golangci-linter.